src/main/java/dev/suvera/keycloak/scim2/storage/events/ScimEventListener.java

```diff
@@ -67,16 +67,17 @@ private void handleUserEvent(AdminEvent event) {
 
             JsonNode representationJson = readJsonString(event.getRepresentation());
 
+            String userId = event.getResourcePath().split("/")[1];
```

Comment:
What happens if there is no value at index 1 - it'll likely fail?

Reply:
I guess we'll get an `IndexOutOfBounds` exception, so it will fail, but what is the situation where this can happen? Likely something is very wrong, and it probably should fail loudly. It's already handled similarly below for group ID.